Javadoc warnings

When running the Javadoc generation 30 warnings are issued:
```
 Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/HeatMapChart.java:90: warning - invalid usage of tag <
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/HeatMapChart.java:90: warning - invalid usage of tag >
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/HeatMapChart.java:90: warning - invalid usage of tag <
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/HeatMapChart.java:90: warning - invalid usage of tag >
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/OHLCChart.java:356: warning - Parameter "xData" is documented more than once.
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/OHLCChart.java:373: warning - Parameter "xData" is documented more than once.
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/OHLCChart.java:390: warning - Parameter "xData" is documented more than once.
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/OHLCChart.java:403: warning - Parameter "xData" is documented more than once.
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/PdfboxGraphicsEncoder.java:71: warning - invalid usage of tag <
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/PdfboxGraphicsEncoder.java:71: warning - invalid usage of tag >
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/PdfboxGraphicsEncoder.java:84: warning - invalid usage of tag <
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/PdfboxGraphicsEncoder.java:84: warning - invalid usage of tag >
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/PdfboxGraphicsEncoder.java:97: warning - invalid usage of tag <
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/PdfboxGraphicsEncoder.java:97: warning - invalid usage of tag >
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/XYChart.java:92: warning - Parameter "xData" is documented more than once.
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/XYChart.java:117: warning - Parameter "xData" is documented more than once.
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java:6: warning - invalid usage of tag &scheme
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java:6: warning - invalid usage of tag &n
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java:6: warning - invalid usage of tag &scheme
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java:6: warning - invalid usage of tag &n
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/HeatMapChart.java:90: warning - invalid usage of tag <
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/HeatMapChart.java:90: warning - invalid usage of tag >
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java:6: warning - invalid usage of tag &scheme
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java:6: warning - invalid usage of tag &n
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/HeatMapChart.java:90: warning - invalid usage of tag <
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/HeatMapChart.java:90: warning - invalid usage of tag >
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java:6: warning - invalid usage of tag &scheme
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java:6: warning - invalid usage of tag &n
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java:6: warning - invalid usage of tag &scheme
Warning:  /home/runner/work/XChart/XChart/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java:6: warning - invalid usage of tag &n
```

--- a/xchart-demo/src/main/java/org/knowm/xchart/demo/charts/area/AreaChart03.java
+++ b/xchart-demo/src/main/java/org/knowm/xchart/demo/charts/area/AreaChart03.java
@@ -11,7 +11,7 @@ import org.knowm.xchart.style.Styler.LegendPosition;
 import org.knowm.xchart.style.markers.SeriesMarkers;
 
 /**
- * Combination Line & Area Chart
+ * Combination of Line and Area Chart
  *
  * <p>Demonstrates the following:
  *

--- a/xchart/src/main/java/org/knowm/xchart/HeatMapChart.java
+++ b/xchart/src/main/java/org/knowm/xchart/HeatMapChart.java
@@ -73,7 +73,7 @@ public class HeatMapChart extends Chart<HeatMapStyler, HeatMapSeries> {
   }
 
   /**
-   * Add a series for a HeatMap type chart using using int arrays
+   * Add a series for a HeatMap type chart using int arrays
    *
    * @param seriesName
    * @param xData the X-Axis data
@@ -87,7 +87,7 @@ public class HeatMapChart extends Chart<HeatMapStyler, HeatMapSeries> {
   }
 
   /**
-   * Add a series for a HeatMap type chart using using List<?>
+   * Add a series for a HeatMap type chart using List&lt;?&gt;
    *
    * @param seriesName
    * @param xData the X-Axis data

--- a/xchart/src/main/java/org/knowm/xchart/OHLCChart.java
+++ b/xchart/src/main/java/org/knowm/xchart/OHLCChart.java
@@ -353,7 +353,7 @@ public class OHLCChart extends Chart<OHLCStyler, OHLCSeries> {
    *
    * @param seriesName
    * @param xData the X-Axis data
-   * @param xData the Y-Axis data
+   * @param yData the Y-Axis data
    * @return A Series object that you can set properties on
    */
   public OHLCSeries addSeries(String seriesName, int[] xData, int[] yData) {
@@ -370,7 +370,7 @@ public class OHLCChart extends Chart<OHLCStyler, OHLCSeries> {
    *
    * @param seriesName
    * @param xData the X-Axis data
-   * @param xData the Y-Axis data
+   * @param yData the Y-Axis data
    * @return A Series object that you can set properties on
    */
   public OHLCSeries addSeries(String seriesName, float[] xData, float[] yData) {
@@ -387,7 +387,7 @@ public class OHLCChart extends Chart<OHLCStyler, OHLCSeries> {
    *
    * @param seriesName
    * @param xData the X-Axis data
-   * @param xData the Y-Axis data
+   * @param yData the Y-Axis data
    * @return A Series object that you can set properties on
    */
   public OHLCSeries addSeries(String seriesName, double[] xData, double[] yData) {
@@ -400,7 +400,7 @@ public class OHLCChart extends Chart<OHLCStyler, OHLCSeries> {
    *
    * @param seriesName
    * @param xData the X-Axis data
-   * @param xData the Y-Axis data
+   * @param yData the Y-Axis data
    * @return A Series object that you can set properties on
    */
   public OHLCSeries addSeries(String seriesName, List<?> xData, List<? extends Number> yData) {

--- a/xchart/src/main/java/org/knowm/xchart/PdfboxGraphicsEncoder.java
+++ b/xchart/src/main/java/org/knowm/xchart/PdfboxGraphicsEncoder.java
@@ -68,7 +68,7 @@ public class PdfboxGraphicsEncoder {
   /**
    * Write multiple charts to a file
    *
-   * @param charts List<? extends Chart>
+   * @param charts List&lt;? extends Chart&gt;
    * @param fileName file name path
    * @throws IOException
    */
@@ -81,7 +81,7 @@ public class PdfboxGraphicsEncoder {
   /**
    * Write multiple charts to a file
    *
-   * @param charts List<? extends Chart>
+   * @param charts List&lt;? extends Chart&gt;
    * @param file File
    * @throws IOException
    */
@@ -94,7 +94,7 @@ public class PdfboxGraphicsEncoder {
   /**
    * Write multiple charts to an OutputStream
    *
-   * @param charts List<? extends Chart>
+   * @param charts List&lt;? extends Chart&gt;
    * @param os OutputStream
    * @throws IOException
    */

--- a/xchart/src/main/java/org/knowm/xchart/XYChart.java
+++ b/xchart/src/main/java/org/knowm/xchart/XYChart.java
@@ -89,7 +89,7 @@ public class XYChart extends Chart<XYStyler, XYSeries> {
    *
    * @param seriesName
    * @param xData the X-Axis data
-   * @param xData the Y-Axis data
+   * @param yData the Y-Axis data
    * @return A Series object that you can set properties on
    */
   public XYSeries addSeries(String seriesName, double[] xData, double[] yData) {
@@ -114,7 +114,7 @@ public class XYChart extends Chart<XYStyler, XYSeries> {
    *
    * @param seriesName
    * @param xData the X-Axis data
-   * @param xData the Y-Axis data
+   * @param yData the Y-Axis data
    * @return A Series object that you can set properties on
    */
   public XYSeries addSeries(String seriesName, float[] xData, float[] yData) {

--- a/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java
+++ b/xchart/src/main/java/org/knowm/xchart/style/colors/BaseSeriesColors.java
@@ -3,7 +3,7 @@ package org.knowm.xchart.style.colors;
 import java.awt.*;
 
 /**
- * Colors are taken from http://colorbrewer2.org/#type=qualitative&scheme=Set3&n=12.
+ * Colors are taken from <a href="http://colorbrewer2.org/#type=qualitative&scheme=Set3&n=12">colorbrewer2.org</a>.
  *
  * @author timmolter
  * @author ekleinod

